etc/Brewfile

```diff
@@ -0,0 +1,11 @@
+# deps for our macos.yml CI job
+
+tap "viamrobotics/brews"
```

Comment:
tapped inside the `config + tab` step.

I don't think it hurts to double-tab. Just wanted to mention in case it was unintentional

Reply:
extra, removed in 7e757689b

rebuild running [here](https://github.com/viamrobotics/rdk/actions/runs/8915431788/job/24485027874)

Reply:
oops I remembered -- need to tap before the cache, bc the tapped formulae are a cache input